iGlance/iGlance/iGlance/UserSettings.swift

```diff
@@ -98,8 +101,12 @@ class UserSettings {
 
     private let userSettingsKey: String = "iGlanceUserSettings"
 
-    init() {
-        settings = loadUserSettings()
+    init(isDefault: Bool) {
```

Comment:
IMO it would be cleaner to add a dedicated `resetSettings` method to this class that overwrites the `settings` variable. Otherwise you always have to provide the constructor with the `isDefault` value.